scripts/install-lansa-base.ps1

```diff
@@ -36,7 +36,7 @@ param (
 Write-Debug "script:IncludeDir = $script:IncludeDir" | Write-Host
 
 function ChocoWait([int] $WaitTimeSeconds = 60) {
-    Write-Host "$(Log-Date) Adding Wait for Choco" | Out-Default
+    Write-Host "$(Log-Date) Adding Wait for Choco" | Write-Host
```

Comment:
`| Write-Host` must be removed

Reply:
Suggested changes are updated here is the commit  [9385d1eb31d5402e962db9fd51773ad0d152ca52](https://github.com/robe070/cookbooks/pull/95/commits/9385d1eb31d5402e962db9fd51773ad0d152ca52)